insights/specs/datasources/env.py

```diff
@@ -0,0 +1,95 @@
+"""
+Custom datasources for environments.
+"""
+import re
+import os
+
+from insights.core.context import HostContext
+from insights.core.dr import SkipComponent
+from insights.core.plugins import datasource
+from insights.core.spec_factory import DatasourceProvider
+
+
+@datasource(HostContext)
+def ld_library_path_of_global(broker):
```

Comment:
The same comment as above, would you rename this method more relevant?

Reply:
Thanks! Updated.